core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiServiceSpec.scala

```diff
@@ -1153,13 +1173,19 @@ class AdminApiServiceSpec extends ApiServiceSpec {
         val groupedWorkflowRecs = workflowRecs.groupBy(_.status)
           .filterKeys((WorkflowStatuses.queuedStatuses ++ WorkflowStatuses.runningStatuses).map(_.toString).contains)
           .mapValues(_.size)
-        val expected = WorkflowQueueStatusByUserResponse(
+        val userOwnerWorkflows = (testData.userOwner.userEmail.value ->
```

Comment:
Some nitpicks: please put the `testUserWorkflows` first and add a comment for the `userOwnerWorkflows` describing that that these are equal to all workflows in the systems besides those of the test user.  Renaming `statusCounts` to something like `testUserStatusCounts` would help to clarify that too.

Since this test would now break if we ever added a third user's workflows to the Default Test Database, this is a good candidate for using `withConstantTestDataApiServices` instead.  We try never to change that one.